scripts/update_torch_wheels.sh

```diff
@@ -5,29 +5,19 @@ set -x
 DIST_BUCKET="gs://tpu-pytorch/wheels"
 TORCH_WHEEL="torch-nightly-cp36-cp36m-linux_x86_64.whl"
 TORCH_XLA_WHEEL="torch_xla-nightly-cp36-cp36m-linux_x86_64.whl"
+TORCHVISION_WHEEL="torch_xla-nightly-cp36-cp36m-linux_x86_64.whl"
```

Comment:
Are my eyes broken or we copying installing the same wheel? 😄

Reply:
Oh wow sorry! My eyes are broken and we're installing the same. Fixed.